system/lib/wasmfs/file.cpp

```diff
@@ -204,7 +211,9 @@ int Directory::Handle::removeChild(const std::string& name) {
     entry->second.file->locked().setParent(nullptr);
     dcache.erase(entry);
   }
-  setMTime(time(NULL));
+  struct timespec ts;
+  clock_gettime(CLOCK_REALTIME, &ts);
+  setMTime(ts);
```

Comment:
It would be good to avoid this repeated code pattern somehow. If we did that then we would have a single place to change if we decided to get time information in another way in the future.

One option might be an `updateMTime` method which would mean "update to the current time". There might be a better name though?

Reply:
For symmetry it might be nice to have it for them all, even if mtime is the most-repeating pattern.

Reply:
Another option might be "updateTime" without mentioning mtime at all. Basically "this file was modified, update whatever time changes need to be done".

Reply:
I thinking making one "update to current time" for each time might be a good option here, since there were a few other spots where only `atime` and `ctime` also had to be set to the current time. Would that be fine for now?